awx/ui_next/src/screens/Credential/CredentialEdit/CredentialEdit.jsx

```diff
@@ -116,7 +116,7 @@ function CredentialEdit({ credential, me }) {
         },
       ] = await Promise.all([
         CredentialTypesAPI.read({ page_size: 200 }),
-        CredentialsAPI.readInputSources(credId, { page_size: 200 }),
```

Comment:
This method only takes one param: https://github.com/ansible/awx/blob/devel/awx/ui_next/src/api/models/Credentials.js#L23

It defaults to a page size of 200

Reply:
@mabashian for the record, there's a hidden performance problem here that I can't see.